### PR TITLE
using logout wrapper

### PR DESCRIPTION
**Pull request by RaniAbboud**
replaced         appIDAuthorizationManager.logout(null, null);
with         appID.logout();
(after adding the new wrapper to the sdk)
updated sdk version in build.gradle

### CODE REVIEW

**Comment by RaniAbboud** [comment]
https://github.ibm.com/security-services/appid-project-management/issues/3235

File: app/build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.github.ibm-cloud-security:appid-clientsdk-android:6.+'
+    implementation 'com.github.ibm-cloud-security:appid-clientsdk-android:6.0.1'
```
Comment (rotembr) [inline app/build.gradle]: why we cant leave it 6.+
it supposed to get lates version which is 6.0.1

Comment (RaniAbboud) [reply app/build.gradle]: I thought that if somehow it gets <6.0.1 we'll get an error that AppID.logout is undefined

Comment (RaniAbboud) [reply app/build.gradle]: changed back to 6.+

**Review by rotembr — APPROVED** [review]

**Pull request merged**